### ISSUE TITLE
Put QR code container to top in pre test page

### ISSUE THREAD

**Issue by mlasak** [cta-wave/dpctf-test-runner#60]
At plugfest in Milan we faced issue that QR codes are cut in some cases on pre test page.
We could fix the problem by setting a fixed size for the QR code (300x300 pixels) but it may be better to show the QR code first and then followed by the title and path to test and other non relevant information for automatic processing

Propose to change order on element and put qr code container to top
[tools/wave/www/pre-test.html](https://github.com/cta-wave/dpctf-test-runner/blob/42e5aaf037c0366f8eba2846b060f5fbbb946983/tools/wave/www/pre-test.html#L14)

**Comment by jpiesing** [comment]
Do you have a picture of one example where the QR code is cut on the test page? I'm having difficulty imaging how this would happen. What element of the HTML page cuts the QR code?

**Comment by mlasak** [comment]
Sure, we have recorded the sessions. Will post image here later...

Basically the problem is not that not another element is cutting/overlaying the qr code, but the qr code itself is floating partly outside the viewport

**Comment by mlasak** [comment]
![IMG_0341](https://github.com/cta-wave/dpctf-test-runner/assets/1204653/073bd763-aca1-4d05-a9b4-f64acce43466)
@jpiesing fyi

**Comment by jpiesing** [comment]
The QR code not being visible or partly outside the viewport may be slightly related to https://github.com/cta-wave/dpctf-test-runner/issues/43.

Isn't it possible to scale the QR code so it fits?

**Comment by mlasak** [comment]
yes, it scaled here to the width, which was not want we wanted. Maybe better to scale it to the height and/or fix the font size of the upper elements and/or put the upper element to the bottom and the qr code first

**Comment by jpiesing** [comment]
> yes, it scaled here to the width, which was not want we wanted. Maybe better to scale it to the height and/or fix the font size of the upper elements and/or put the upper element to the bottom and the qr code first

Why not scale it to the height?

**Comment by mlasak** [comment]
Sure, scaling to height could work as well. Much space is consumed by title "Next test is about to start" etc. hope qr code scaled to height is still readable when scaled to height
